sirepo/raydata/scan_monitor.py

```diff
@@ -252,6 +309,13 @@ def _text_query(search_text):
             s.status = d.get(s.uid, _AnalysisStatus.NONE)
         return l, pc
 
+    def _is_float(self, value):
```

Comment:
This name is a little misleading, I think `_is_float_or_float_string` or `_can_float` / `_can_cast_to_float` would be clearer. I saw the comment above where it's used but I think it's better to move that documentation into in the function name or docstring. I also think a more specific exception (ValueError) would be an improvement.

Reply:
I've made some of the changes above with 65cd9d7
- layout changes to the search field selection and add column selector
- renamed _is_float() to _can_cast_to_float() and catch ValueError only

I couldn't easily fix the case sensitivity difference because the standard mongo $text indexed case insensitive while the individual field search can't be changed to case insensitive without changing it to use regular expressions, which I think might slow it down.

The "Add Column" is now bold, but I couldn't modify the margins too much because the "select" input field really wants to size to the maximum text value.